src/main/java/storm/sqs/SqsQueueSpout.java

```diff
@@ -68,6 +68,7 @@
 	public SqsQueueSpout(String queueUrl, boolean reliable) {
 		this.queueUrl = queueUrl;
 		this.reliable = reliable;
+		this.queue = new LinkedBlockingQueue<>();
```

Comment:
Sounds good, I have not tested this on remote cluster and this works fine on local cluster. I will remove the keyword ```final``` so that worker can create in ```open()```